rskj-core/src/main/java/org/ethereum/vm/PrecompiledContracts.java

```diff
@@ -57,11 +58,14 @@
     public static final String SAMPLE_ADDR_STR = "0000000000000000000000000000000001000005";
     public static final String BRIDGE_ADDR_STR = "0000000000000000000000000000000001000006";
     public static final String REMASC_ADDR_STR = "0000000000000000000000000000000001000008";
+    public static final String BTOUTILS_ADDR_STR = "0000000000000000000000000000000001000009";
```

Comment:
The same Address is used in https://github.com/rsksmart/rskj/blob/blockheader_contract/rskj-core/src/main/java/org/ethereum/vm/PrecompiledContracts.java#L63  please talk with @diegomasini so you don't colide

Reply:
Good catch! Talked with @diegomasini and we're keeping this address here.